src/modules/auto-invisible-recaptcha.js

```diff
@@ -25,35 +28,45 @@
 	var loaded = false;
 	var ids = 0;
 
-	var load = function (ctn) {
-		if (!loaded || !ctn) {
+	var load = function () {
+		if (!loaded || !page) {
 			return;
 		}
-		ctn.find(options.trigger).each(function () {
+		page.find(options.trigger).each(function () {
 			var t = $(this);
 			if (t.attr('id')) {
 				return;
 			}
 			var id = options.prefix + (++ids);
 			t.attr('id', id);
-			var recaptchaId = window.grecaptcha.render(id, {
+			var wid = window.grecaptcha.render(id, {
 				sitekey: t.attr('data-sitekey'),
+				badge: t.attr('data-badge') || 'inline',
```

Comment:
We should use Google's default value instead of inline: bottomright

Reply:
Google's default sucks ass. It always gets mixed up with our designs.

Reply:
Yeah but I always get the comment to put it back their because the inline style does not match the esthetics of the website.

Reply:
@PascalPiche ?

Reply:
Prefer to get it at the wrong place forcing the Dev to make sure it will be correctly visible when put with the default value : bottomright. Can we add documentation of available options in the header please :)